src/stories/layout-helpers/StyledVerticalStack.tsx

```diff
@@ -0,0 +1,15 @@
+import { PropsWithChildren } from 'react';
+
+export interface StyledVerticalStackProps extends PropsWithChildren {
+  gap?: number;
+}
+
+export default function StyledVerticalStack({ children, gap = 0 }: StyledVerticalStackProps) {
+  const gapRem = gap / 4;
+  const spacing = gapRem + 'rem';
```

Comment:
Could also use string interpolation here.

Reply:
as stated earlier, i created an external function for all these calculations ...